src/selfplay/game.cc

```diff
@@ -60,6 +61,9 @@ const OptionId kSyzygyTablebaseId{
     "List of Syzygy tablebase directories, list entries separated by system "
     "separator (\";\" for Windows, \":\" for Linux).",
     's'};
+const OptionId kOpeningStopProbId{"opening-stop-prob", "OpeningStopProb",
```

Comment:
Alternatively, we can have an inverse of this as a parameter, which is the average early stop move.
`AverageOpeningCutoffMove` or something like this.

Hmmmm although if tossed both for white and black, it's not that easy..